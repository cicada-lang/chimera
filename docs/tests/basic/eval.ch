let h = quote g(y)

print h
print eval h

print quote f(x, eval h)
