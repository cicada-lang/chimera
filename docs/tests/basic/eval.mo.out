g(y)
g(y)
f(x, g(y))
